frontend/public/components/console-notifier.tsx

```diff
@@ -19,7 +19,8 @@ const ConsoleNotifier_: React.FC<ConsoleNotifierProps> = ({obj, location}) => {
         style={{
           backgroundColor: notification.spec.backgroundColor,
           color: notification.spec.color,
-        }}>
+        }}
+        data-test={`${notification.metadata.name}-${notification.spec.location}`}>
```

Comment:
nit: I have a slight preference for `data-test-id` here and below

Reply:
I opted for `data-test` here because `id` implies it is unique, and it could be duplicated if `position: BannerTopBottom`, but we're not currently testing that value.

I opted for `data-test` below as `data-test=user-dropdown` is already in use in the file. 

Happy to change both if you feel strongly.